manifests/certonly.pp

```diff
@@ -2,6 +2,46 @@
 #
 # This type can be used to request a certificate using the `certonly` installer.
 #
+# @example standalone authenticator
+#   letsencrypt::certonly { 'foo.example.com': }
+#
+# @example Apache authenticator
+#   letsencrypt::certonly { 'foo':
+#     domains => ['foo.example.com', 'bar.example.com'],
```

Comment:
While we're at it, I'd also consider splitting it:
```suggestion
# @example Multiple domains
#   letsencrypt::certonly { 'foo':
#     domains => ['foo.example.com', 'bar.example.com'],
#
# @example Apache authenticator
#   letsencrypt::certonly { 'foo.example.com':
```

This could be applied to all examples to keep them shorter. Again, a bit offtopic so feel free to ignore it for this PR.

Reply:
https://github.com/voxpupuli/puppet-letsencrypt/pull/267/commits/907aae5353c235b2eaeafd65ffa237d8ed64d56d